hook.php

```diff
@@ -235,8 +235,8 @@ function plugin_singlesignon_install() {
                   `split_name`                 tinyint(1) NOT NULL DEFAULT '0',
                   `is_deleted`                 tinyint(1) NOT NULL default '0',
                   `comment`                    text COLLATE utf8_unicode_ci,
-                  `date_mod`                   datetime DEFAULT NULL,
```

Comment:
Why this change?

Reply:
Because of this message after activating the plugin:

![grafik](https://github.com/edgardmessias/glpi-singlesignon/assets/66548244/7bc49293-b88a-4df9-a337-70c99e27a341)

I always need to run the command (`php bin/console migration:timestamps`) described there. In the database schema I saw that this command change the type of the 2 columns to datetime. So I decided to change the type directly in the create table. Maybe it depends on the glpi version. I'm working on version 10.0.14.